includes/wikia/services/WikiService.class.php

```diff
@@ -841,6 +852,22 @@ public function getDetails( Array $wikiIds = null ) {
 					'city_visualization' => array(
 						'LEFT JOIN',
 						'city_list.city_id = city_visualization.city_id'
+					),
+					'city_verticals' => array(
+						'LEFT JOIN',
+						'city_list.city_vertical = city_verticals.vertical_id'
+					),
+					'city_domains' => array(
+						'LEFT JOIN',
+						'city_list.city_id = city_domains.city_id'
+					),
```

Comment:
`city_list.city_url` column keeps the "primary" wiki domain. Maybe it will be enough here to just use this column instead of a join.